### ISSUE TITLE
Can't install with Laravel 9

### ISSUE THREAD

**Issue by Stajor** [j0k3r/php-readability#76]
```
Problem 1
    - Root composer.json requires j0k3r/php-readability ^2.0 -> satisfiable by j0k3r/php-readability[2.0.0, 2.0.1].
    - j0k3r/php-readability[2.0.0, ..., 2.0.1] require psr/log ^1.0 -> found psr/log[1.0.0, ..., 1.1.4] but the package is fixed to 3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
```

**Comment by j0k3r** [comment]
Too bad, but without the full composer, it's impossible to fix.

**Comment by DerFichtl** [comment]
You using psr/log ^1.0 (major version fixed). But the current version is already 3.0 and many other packages don't support psr/log 1.0 anymore. So there is always a version conflict between php-readability and other packages.

Laravel seems to have it fixed and still support psr/log 1.0 but maybe you can consider to just change the dependency to: 
"psr/log": "^1.0|^2.0|^3.0" ?
